### PR TITLE
Added implementation

### PR DESCRIPTION
**Pull request by oleksandrdziubiuk**

### CODE REVIEW

File: src/main/java/com/dev/cinema/model/dto/UserRequestDto.java [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+import com.dev.cinema.lib.ValidEmail;
+import com.dev.cinema.lib.ValidPassword;
+import javax.validation.constraints.Min;
+
+@ValidPassword(field = "password", fieldMatch = "repeatPassword")
 public class UserRequestDto {
+    @ValidEmail
     private String email;
+    @Min(6)
     private String password;
```
Comment (engineer027) [inline src/main/java/com/dev/cinema/model/dto/UserRequestDto.java]: wrappers are not supported by this annotation

**Review by engineer027 — APPROVED** [review]

File: src/main/java/com/dev/cinema/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                .httpBasic()
+                .and()
+                .csrf().disable();
+
+        http
+                .logout()
+                .permitAll()
+                .logoutUrl("/logout")
+                .logoutSuccessUrl("/login")
+                .invalidateHttpSession(true);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/config/SecurityConfig.java]: isn't this the default behaviour?

File: src/main/java/com/dev/cinema/lib/PasswordValidator.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    @Override
+    public boolean isValid(Object object, ConstraintValidatorContext context) {
+        Object passwordValue = new BeanWrapperImpl(object)
+                .getPropertyValue(password);
+        Object repeatPasswordValue = new BeanWrapperImpl(object)
+                .getPropertyValue(repeatPassword);
+        if (passwordValue == null) {
+            return false;
+        }
+        return passwordValue == repeatPasswordValue;
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/lib/PasswordValidator.java]: it is the right way to compare Strings?

File: src/main/java/com/dev/cinema/lib/ValidPassword.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+    Class<?>[] groups() default {};
+
+    Class<? extends Payload>[] payload() default {};
+
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        ValidPassword[] value();
+    }
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/lib/ValidPassword.java]: do you need this?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**